{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,3 +16,19 @@ universal = 1
 
 [flake8]
 exclude = docs
+
+[aliases]
+{%- if cookiecutter.use_pytest == 'y' %}
+test = pytest
+{%- endif %}
+# Define setup.py command aliases here
+{%- if cookiecutter.use_pytest == 'y' %}
+
+[pytest]
+# Set permanent options for the "pytest" setup command here
```

Comment:
``[pytest]`` is for the “pytest” distutils command, served by the pytest-runner setuptools extension.  pytest itself historically used the same section, but last year it started using ``[tools:pytest]`` instead – after the pytest-runner author complained in pytest-dev/pytest#567 that pytest itself, not being a distutils command, should not occupy a section whose name is not colon-prefixed.